src/Sunny.jl

```diff
@@ -89,14 +92,30 @@ include("StructureFactors/StructureFactors.jl")
 include("StructureFactors/SFUtils.jl")
 include("StructureFactors/SampleGeneration.jl")
 include("StructureFactors/FormFactor.jl")
-include("StructureFactors/ElementContraction.jl")
 include("StructureFactors/BasisReduction.jl")
-include("StructureFactors/Interpolation.jl")
-include("StructureFactors/PowderAveraging.jl")
 include("StructureFactors/DataRetrieval.jl")
 export DynamicStructureFactor, InstantStructureFactor, StructureFactor, FormFactor, 
-    add_sample!, intensities, instant_intensities, broaden_energy, lorentzian,
-    connected_path, all_exact_wave_vectors, ωs, spherical_shell, merge!
+    add_sample!, broaden_energy, lorentzian,
+    all_exact_wave_vectors, ωs, spherical_shell, merge!, intensity_formula, integrated_lorentzian
+
+include("Intensities/ElementContraction.jl")
+
+include("Intensities/Interpolation.jl")
+export intensities_interpolated, instant_intensities_interpolated, connected_path
+
+include("Intensities/Binning.jl")
+export intensities_binned, instant_intensities_interpolated, connected_path_bins,
+    BinningParameters, integrate_axes!, unit_resolution_binning_parameters,
+    rlu_to_absolute_units!,
+    intensities_binned, slice_2D_binning_parameters, axes_bincenters,
+    connected_path_bins, count_bins
+
+
+include("Intensities/PowderAveraging.jl")
+export powder_averaged_bins, powder_average
+
+include("Intensities/ExperimentData.jl")
+export load_nxs, generate_mantid_script_from_binning_parameters
```

Comment:
This is a long list of exports. Have all of them been carefully considered? (I'm not familiar with these functions in particular, so please take this as a general comment).

Context: The export list is part of the package's SemVer guarantee. That is, there is an implicit promise not to change the behavior of functions if they are exported. If any of these functions still be evolving, we can hide them from the export list, while access is still possible through the syntax `Sunny.my_unexported_function`.

Reply:
> Context: The export list is part of the package's SemVer guarantee. That is, there is an implicit promise not to change the behavior of functions if they are exported. If any of these functions may still be evolving, we can hide them from the export list, while access is still possible through the syntax `Sunny.my_unexported_function`.

I've just gone through and cleaned up the exports a bit. In terms of SemVer guarantee, we need to settle the units of the `BinningParameters` before unleashing this on the world, but once it is settled, `BinningParameters`, `count_bins`, `axes_bincenters`, `rlu_to_absolute_units!` (or its replacement)

Reply:
@ddahlbom I think that's right re: `powder_average_interpolated`, since that function can (and is) implemented in terms of `intensities_interpolated`, whereas `powder_average_binned` is impossible to implement correctly in terms of `intensities_binned`. I vote un-export `powder_average_interpolated` but mention its existence in the Example

Reply:
I'm fine with not exporting it. (Reference to it should then be eliminated from the doc string for the binned version.)